content/TrueCommand/TCGettingStarted/Install/InstallTCCloud.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Installing TrueCommand Cloud"
+description: "This article describes the steps for installing TrueCommand Cloud."
+weight: 10
+tags:
+- tccloudinstall
+- tcinstall
+- tccloud
+- scaletoptoolbar
+- coretoptoolbar
+---
+{{< toc >}}
+
+TrueCommand is versatile and offers several different install options.
+TrueCommand Cloud is the preferred method for using TrueCommand since it requires no local resources or specific hardware requirements to get started!
+
+{{< hint info >}}
+A credit card number is required for Cloud Signups.
+
+If the TrueCommand Cloud license was purchased as part of an Enterprise contract, please contact [**iXSystems Customer Support**]({{< relref "/TrueCommand/Introduction/Support.md" >}}) and a member of the Deployment Team will provide a sign-up code which makes the cost $0 for the duration of the contract.
+{{< /hint >}}
+
+## Begin TrueCommand Cloud Installation
+
+{{< expand "TrueCommand Cloud: Register and Create Subscription" >}}
```

Comment:
Couple of things:
Starting an expand under a heading that has other subheadings under it means those headings do no get listed in the TOC so we need to be careful where we use expands. This is also the reason we did away with tabs because headings in tabs also do not get listed in the TOC.
Too many headings followed immediately by expands chops up the article too much and doesn't improve readability of the article. We want to use expands to collapse long procedures that advanced users might want to skip over if they know how to do that part of the procedure.
When doing the expand shortcode, Tim told me to include the "v" as the expand down instruction for the shortcode.
```suggestion
{{< expand "TrueCommand Cloud: Register and Create Subscription" "v" >}}
```

Reply:
Added "v"